src/want-manager/index.js

```diff
@@ -111,24 +112,25 @@ module.exports = class WantManager {
     this._stopPeerHandler(peerId)
   }
 
-  run () {
+  start (callback) {
+    // resend entire wantlist every so often
     this.timer = setInterval(() => {
-      // resend entirew wantlist every so often
+      this._log('resend full-wantlist')
       const fullwantlist = new Message(true)
-      for (let entry of this.wantlist.entries()) {
-        fullwantlist.addEntry(entry[1].cid, entry[1].priority)
-      }
-
-      this.peers.forEach((p) => {
-        p.addMessage(fullwantlist)
+      this.wantlist.forEach((entry) => {
+        fullwantlist.addEntry(entry.cid, entry.priority)
       })
-    }, 10 * 1000)
+
+      this.peers.forEach((p) => p.addMessage(fullwantlist))
+    }, 60 * 1000)
```

Comment:
We already generate a lot of traffic, so I would rather go to a low value and increase it if we see issues with it